src/app/charts/content/templates/ark-secret-conn.yaml

```diff
@@ -22,6 +22,22 @@
     {{- $data = set $data "group" ($groups | b64enc) }}
     {{- $data = set $data "url" ($content.url | b64enc) }}
     {{- $data = set $data "ui" ($content.ui | b64enc) }}
+
+    {{- if (eq $content.dialect "box") }}
```

Comment:
From https://gitlab.armedia.com/arkcase/acm-config/-/merge_requests/10310, I see that there are other values that also need to be here:

- clientId
- clientSecret
- enterpriseId
- publicKeyId
- privateKeyPassword

These values should probably live in `global.subsys.content.settings` instead of `global.env` as is currently implemented.

This also means that those values must also be consumed by the `core` chart's `subsys-deps.yaml`